### ISSUE TITLE
mcpi - may not work with python 3.10

### ISSUE THREAD

**Issue by denkigish** [martinohanlon/mcpi#30]
I have noticed that if I use python 3.10, that I get errors when running scripts using the mcpi library, however it works with 3.9.2.  Last time I looked at it I mentioned something bout iterates.

**Comment by martinohanlon** [comment]
I believe this has already been addressed. See PR https://github.com/martinohanlon/mcpi/pull/21/commits/2e8112af6389563e645509b2690dd45000f1c907

Are you running the latest version from pypi.

**Comment by denkigish** [comment]
I am running version 1.2.1 of mcpi.
It runs just fine on 3.9.2 and 3.9.6
I use pyenv.
<img width="770" alt="Screen Shot 2022-12-31 at 8 15 32 AM" src="https://user-images.githubusercontent.com/64050980/210140166-5bc99307-737a-425f-ba0c-165e21eb8827.png">

**Comment by josephruscio** [comment]
FWIW just hit the same thing with 3.11, working on installing 3.9 from sources (binary installers no longer provided) but given the above and [this reddit thread](https://www.reddit.com/r/learnpython/comments/ungfxm/minecraft_programming_error/) I'm optimistic on that as a workaround.

**Comment by josephruscio** [comment]
Just following up to confirm that moving to an earlier version of Python (3.9.x) works for me as well.

**Comment by martinohanlon** [comment]
I have been through and tested this and I cant reproduce the error running mcpi v.1.2.1 and Python 3.10+

@denkigish your error message suggests you are running an older version of mcpi

line 5 of `util.py` is no longer `if is instance ...` 

You can see the new line 5 here - https://github.com/martinohanlon/mcpi/blob/master/mcpi/util.py#L5

However back in v.1.2, line 5 was `if is instance` - https://github.com/martinohanlon/mcpi/commit/3482819d43c24e0ae8713b2dc596c73f6e2d874e#diff-e72d30878248ed9fd5808f36dc652597e3633cfc2c9795fd1d9cdc7412d012ebL5

So, in summary I think you need to upgrade the version you are running.

```
pip3 install mcpi --upgrade
```
